config/routes.rb

```diff
@@ -53,6 +53,11 @@
           end
         end
         resources :starter_file_groups, only: [:index, :create]
+        resources :submission_files, only: [] do
+          collection do
+            post 'submit_file'
```

Comment:
In terms of organization, I think this route should go under `assignments` (as a member route). The functionality is more than just submission files, since it has the potential to create a group as well.

(Also update Changelog entry accordingly.)